classes/Http/Controller/ForgotController.php

```diff
@@ -211,7 +211,9 @@ public function updatePasswordAction(Request $request)
         $form->handleRequest($request);
         
         if (!$form->isValid()) {
-            return $this->render('user/reset_password.twig', ['form' => $form->createView()]);
```

Comment:
While at it, I figured it would make sense to streamline how we pass data to the templates - this is an example.